src/clients/repositories.rb

```diff
@@ -1939,6 +1944,16 @@ def StartInstSource
       UI.CloseDialog
       ret
     end
+
+    def warn_service_repository(source_state)
+      msg = _("Repo %{name} is managed by service %{service}.\n"\
+        "Volatile changes are reset by the next service refresh!") % {name: source_state["name"],
```

Comment:
I'd rephrase this part of the message "Your manual changes will be reset by the next service refresh!"